tests/common/common_python_vec.py

```diff
@@ -219,9 +219,9 @@ class ReverseData:
 }
 """)
 
-def clear_type_str(type_str):
+def remove_namespaces_whitspaces(type_str):
     """
-    Clear type name from from namespaces ans whitspaces
+    Clear type name from namespaces and whitspaces
```

Comment:
I guess you mean `whitespaces` or `white_spaces` everywhere in the code and comments.

Reply:
Yes, missed it, thank you! fixed.